playwright/src/main/java/com/microsoft/playwright/impl/Utils.java

```diff
@@ -457,4 +457,29 @@ private static String base64Buffer(byte[] bytes, Path path) throws IOException {
     }
     return Base64.getEncoder().encodeToString(bytes);
   }
+
+  static JsonObject interceptionPatterns(List<UrlMatcher> matchers) {
+    JsonArray jsonPatterns = new JsonArray();
+    for (UrlMatcher matcher: matchers) {
+      JsonObject jsonPattern = new JsonObject();
+      Object urlFilter = matcher.rawSource;
+      if (urlFilter instanceof String) {
+        jsonPattern.addProperty("glob", (String) urlFilter);
+      } else if (urlFilter instanceof Pattern) {
+        Pattern pattern = (Pattern) urlFilter;
+        jsonPattern.addProperty("regexSource", pattern.pattern());
+        jsonPattern.addProperty("regexFlags", toJsRegexFlags(pattern));
+      } else {
+        // Match all requests.
+        jsonPattern.addProperty("glob", "**/*");
```

Comment:
Do you want to remove all other properties added prior to this point? It should not matter technically, but would be cleaner.

Reply:
There should be no properties as the jsonPatter was just created at line 464.